pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -32,18 +34,44 @@ const (
 	mountNamespace = "/proc/1/ns/mnt"
 )
 
+// GetHostNamespacePath - returns the path of the parent process's mount namespace.
+// This is needed to handle Kubernetes and KubeVirt running inside a container,
+// e.g., in a containerd-based runtime, where the mount namespace is different
+// from the host/node.
+func GetHostNamespacePath() string {
+	return fmt.Sprintf("/proc/%d/ns/mnt", os.Getppid())
+}
+
 func getBaseArgs() []string {
-	return []string{"--mount", mountNamespace}
+	return []string{"--mount", GetHostNamespacePath()}
```

Comment:
Should this be `GetChrootMountNamespace` ?

Reply:
@xpivarc you are right. updated. Thanks.